app/uk/gov/hmrc/helptosavefrontend/views/contact_hmrc.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.helptosavefrontend.config._
+@import uk.gov.hmrc.helptosavefrontend.helpers.pageTitle
+@import uk.gov.hmrc.play.frontend.auth.AuthContext
+@import uk.gov.hmrc.helptosavefrontend.models.HtsContext
+@import uk.gov.hmrc.helptosavefrontend.config.PartialRetriever
+
+@(partialUrl: String, formBody: Option[Html], formProvider: PartialRetriever)(implicit request: Request[_], lang: Lang, messages: Messages, htsContext: HtsContext)
```

Comment:
`partialUrl` is only there in order to be passed in to the `formProvider` so it almost doesn't need to be there. The type signature could be:
```
@(formBody: Option[Html], backupFormBody: => Html) 
```
and the code calling it would look like:
```scala
views.html.contact_hmrc(None, formProvider.getPartialContent("..."))
```